### ISSUE TITLE
Confirmation on "New", "Generate", and "Clear" buttons

### ISSUE THREAD

**Issue by rahji** [keiranking/Phil#37]
It's way too easy to touch one of these buttons accidentally and wipe everything out. A confirmation would be a simple way to resolve this.

**Comment by orion** [comment]
Agreed!  You could even do it only when there is any data in the grid or changes to the generated layout.  

Also: what is the function of the "freeze grid" button, and why is it never active for me?
